private/package.json

```diff
@@ -15,7 +15,8 @@
     "watch": "webpack --mode development --watch",
     "lint:scripts": "eslint ./src/",
     "lint:styles": "stylelint \"src/**/*.scss\"",
-    "lint": "yarn lint:scripts; yarn lint:styles"
+    "lint": "yarn lint:scripts; yarn lint:styles",
+    "env": "npx @wordpress/env"
```

Comment:
Given that the project uses `yarn` instead of `npm`, should`npx` be replaced by the equivalent `yarn dlx` ?

Reply:
Replaced in 96ff301c4260bfae47ba0f507c53ea4b7bab88d7